ct1/vehinhtron.cpp

```diff
@@ -0,0 +1,79 @@
+#include<iostream>
+
+using namespace std;
+
+string bankinh, toadox, toadoy;
+ int x, y, r;
+
+int check(int a, int b){
+	int tmp;
+	tmp = (a-x)*(a-x)+ (b-y)*(b-y) - r*r;
+	if(tmp<=0) return 1;
+	return 0;	
+}
+void in(){
+	int i =x; int j =y;
```

Comment:
i, j là biến anonymous, chỉ sử dụng nội bộ bên trong vòng for thì không cần khai báo ngoài như thế này.